_layouts/downloadpage.html

```diff
@@ -79,6 +83,9 @@ <h3>Then, install Scala:</h3>
             <div class="text-step">
               <h2>Other ways to install Scala</h2>
               <ul>
+                <li>
+                  Follow <a href="https://docs.scala-lang.org/getting-started/index.html#or-manually">the getting-started</a> for a manual installation.
```

Comment:
It is not clear to me what “manual” installation means. I guess you mean a setup with a text editor and command-line tools (no IDE)?

Reply:
Also, it seems that this page _is_ the getting started, right?

Reply:
yes, it's the getting started.

Reply:
So, what I mean is that we are already in the “getting started” page and we talk about another “getting started” page, so I think this is a bit confusing. Furthermore, I don’t think “manual” installation is very clear.

Reply:
ok. I will remove this..

Reply:
I think it is useful to still keep a link to setup instructions that are not IDE-based. I just don’t think the phrasing is clear here.